src/Ulid.php

```diff
@@ -87,4 +92,58 @@ public function __toString()
     {
         return $this->string();
     }
+
+    //"0123456789ABCDEFGHJKMNPQRSTVWXYZ"
+    protected static function crockford()
```

Comment:
I am not fan of static methods. To avoid instantiating the base32 encoder twice it could be instantiated in the constructor instead.

```php
private $base32;
...
$this->base32 = new \Tuupola\Base32([
    "characters" => \Tuupola\Base32::CROCKFORD,
    "padding" => false
]);
```

And then it can be used later as:

```php
$encoded = $this->base32->encode($this->timestamp);
```